src/main/scala/cromwell/engine/workflow/WorkflowOptions.scala

```diff
@@ -97,17 +108,24 @@ case class WorkflowOptions(jsObject: JsObject) {
 
   def toMap = jsObject.fields
 
-  def get(key: String): Try[String] = jsObject.fields.get(key) match {
-    case Some(jsStr: JsString) => Success(jsStr.value)
-    case Some(jsObj: JsObject) if isEncryptedField(jsObj) => decryptField(jsObj)
-    case Some(jsVal: JsValue) => Failure(new IllegalArgumentException(s"Unsupported value as JsValue: $jsVal"))
-    case None => Failure(new Throwable(s"Field not found: $key"))
-  }
+  def get(key: String): Try[String] = WorkflowOptions.get(key, jsObject)
 
   def getBoolean(key: String): Try[Boolean] = jsObject.fields.get(key) match {
     case Some(jsBool: JsBoolean) => Success(jsBool.value)
     case Some(jsVal: JsValue) => Failure(new IllegalArgumentException(s"Unsupported JsValue as JsBoolean: $jsVal"))
-    case None => Failure(new Throwable(s"Field not found: $key"))
+    case None => Failure(new OptionNotFoundException(s"Field not found: $key"))
+  }
+
+  def getDefaultRuntimeOption(key: String): Try[String] = jsObject.fields.get(defaultRuntimeOptionKey) match {
+    case Some(jsObj: JsObject) => WorkflowOptions.get(key, jsObj)
+    case Some(jsVal) => Failure(new IllegalArgumentException(s"Unsupported JsValue for $defaultRuntimeOptionKey: $jsVal. Expected a JSON object."))
+    case None => Failure(new OptionNotFoundException(s"Field not found: $key"))
+  }
+
+  def getDefaultRuntimeOptionKeys: Iterable[String] = jsObject.fields.get(defaultRuntimeOptionKey) match {
+    case Some(jsObj: JsObject) => jsObj.fields.keys
+    case _ => Seq.empty[String]
```

Comment:
We should stop making Seqs as concrete entities as they really are a List and it obscures that fact.